### PR TITLE
AP-1040 content and style changes

### PR DESCRIPTION
**Pull request by jim-laney**
https://dsdmoj.atlassian.net/browse/AP-1040

- changed bulleted list to numbered list
- added third item to list ('keep it on file')
- changed feedback sentence and made it inset text

### CODE REVIEW

File: app/views/providers/saml_idp/idp/saml_post.html.erb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    <meta http-equiv='X-UA-Compatible' content='IE=edge,chrome=1'>
+  </head>
+  <body onload="document.forms[0].submit();">
+    <%= form_tag("#{Rails.configuration.x.application.host_url}/providers/saml/auth") do %>
+      <%= hidden_field_tag('SAMLResponse', @saml_response) %>
+      <%= hidden_field_tag('RelayState', params[:RelayState]) %>
+      <%= submit_tag 'Submit' %>
+    <% end %>
+  </body>
+</html>
```
Comment (mpw5) [inline app/views/providers/saml_idp/idp/saml_post.html.erb]: Did you mean to add this file? I'm not sure what it's doing! If not you'll need to delete it

Comment (jim-laney) [reply app/views/providers/saml_idp/idp/saml_post.html.erb]: Hi Mark. No, I didn't. I didn't touch that file but it somehow ended up as part of the commit. Where do I delete it?

**Comment by mpw5** [comment]
In your editor you should be able to see the file at `app/views/providers/saml_idp/idp/saml_post.html.erb`. You can just delete it from there. Then just commit again and push.

File: app/views/providers/saml_idp/idp/saml_post.html.erb (hunk 1 above)
Comment (mpw5) [reply app/views/providers/saml_idp/idp/saml_post.html.erb]: In your editor you should be able to see the file at app/views/providers/saml_idp/idp/saml_post.html.erb. You can just delete it from there. Then just commit again and push.

Comment (jim-laney) [reply app/views/providers/saml_idp/idp/saml_post.html.erb]: OK (I think) I've done that

**Pull request merged**